app/src/test/java/com/lokico/PSWind/BasicUnitTests.java

```diff
@@ -0,0 +1,30 @@
+package com.lokico.PSWind;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * To work on unit tests, switch the Test Artifact in the Build Variants view.
+ */
+public class BasicUnitTests {
+    @Test
+    public void isNoaaImplemented() {
+        assertEquals(true, NOAA.isActivityImplemented());
```

Comment:
You can use assertTrue() instead of assertEquals() here.  In this situation, it's just more succinct.  It's actually more important to use assertEquals() instead of doing something like:
`assertEquals(true, NOAA.ManyOverlaysAreAvailable() == 3`

That makes the failure message actually helpful.